hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/InfrastructureFactory.java

```diff
@@ -240,7 +240,10 @@ public TransferLogic newTransferLogic(
                 recordsHistorian,
                 txnCtx,
                 aliasManager,
-                feeDistribution);
+                feeDistribution,
+                // No-op, we won't have used any frontend throttle capacity for an auto-creation
+                // attempted during an EVM transaction
+                n -> {});
```

Comment:
Why is this not possible, can it not trigger a `CryptoTransfer`? Is that because we reduce gas instead?

Reply:
You're correct this _could_ do a `CryptoTransfer` with an auto-creation; but since the frontend `ContractCall` throttle won't have taken that into account, we don't want to refund any corresponding capacity to the frontend throttle bucket.